api/src/test/java/org/openmrs/module/fhir2/api/translators/impl/AllergyIntoleranceReactionComponentTranslatorImplTest.java

```diff
@@ -10,10 +10,7 @@
 package org.openmrs.module.fhir2.api.translators.impl;
 
 import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.Matchers.equalTo;
-import static org.hamcrest.Matchers.greaterThanOrEqualTo;
-import static org.hamcrest.Matchers.notNullValue;
-import static org.hamcrest.Matchers.nullValue;
+import static org.hamcrest.Matchers.*;
```

Comment:
I think this is the last thing to clean-up, but we [specifically do not allow `*` imports](https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Donotusewildcardimports)